*** tests/A-007-007-M-ok.out	2023-10-12 05:38:21.567711831 +0100
--- tests/A-007-007-M-ok.outhyp	2023-10-12 12:24:55.512187251 +0100
***************
*** 1,34 ****
- Menu Principal
- 1 - Novo
- 2 - Abrir
- 3 - Guardar
- 4 - Menu de Edição
- 5 - Menu de Consultas
- 0 - Sair
- Escolha uma opção: Menu de Edição
- 1 - Visualizar
- 2 - Inserir
- 3 - Copiar
- 4 - Apagar
- 5 - Cortar
- 6 - Colar
- 7 - Mostrar 'cut buffer'
- 0 - Sair
- Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Visualizar: Operação inválida: A gama '1;1:5;1' é inválida.
- Menu de Edição
- 1 - Visualizar
- 2 - Inserir
- 3 - Copiar
- 4 - Apagar
- 5 - Cortar
- 6 - Colar
- 7 - Mostrar 'cut buffer'
- 0 - Sair
- Escolha uma opção: Menu Principal
- 1 - Novo
- 2 - Abrir
- 3 - Guardar
- 4 - Menu de Edição
- 5 - Menu de Consultas
- 0 - Sair
- Escolha uma opção: 
\ No newline at end of file
--- 0 ----
