*** tests/A-014-014-M-ok.out	2023-10-12 02:53:40.102737818 +0100
--- tests/A-014-014-M-ok.outhyp	2023-10-12 06:20:44.830161446 +0100
***************
*** 14,23 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 1;1|5
! 1;2|49
! 1;3|5=1;1
! Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
--- 14,20 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
