<<<<<<< HEAD
*** tests/A-103-103-M-ok.out	2023-10-23 16:29:35.429801666 +0100
--- tests/A-103-103-M-ok.outhyp	2023-10-26 05:41:05.322398410 +0100
=======
*** tests/A-103-103-M-ok.out	2023-10-26 00:06:55.674645967 +0100
--- tests/A-103-103-M-ok.outhyp	2023-10-26 06:01:42.283584599 +0100
>>>>>>> main
***************
*** 14,20 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 2;3|#VALUE=ADD(2;20,5)
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 14,20 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 2;3|5=ADD(2;20,5)
  Menu de Edição
  1 - Visualizar
  2 - Inserir
***************
*** 24,30 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 3;3|#VALUE=MUL(4;1,4;10)
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 24,30 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 3;3|0=MUL(3,0)
  Menu de Edição
  1 - Visualizar
  2 - Inserir
