*** tests/A-107-107-M-ok.out	2023-10-29 09:20:55.325976181 +0000
--- tests/A-107-107-M-ok.outhyp	2023-10-29 09:35:09.559767355 +0000
***************
*** 28,35 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Insira o conteúdo da célula: Inserir: Operação inválida: A função 'MAXX' é desconhecida.
! Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
--- 28,34 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Insira o conteúdo da célula: Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
