*** tests/A-119-119-M-ok.out	2023-10-29 11:35:19.105282324 +0000
--- tests/A-119-119-M-ok.outhyp	2023-10-29 11:39:14.483235640 +0000
***************
*** 109,128 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
- Escolha uma opção: Menu de Edição
- 1 - Visualizar
- 2 - Inserir
- 3 - Copiar
- 4 - Apagar
- 5 - Cortar
- 6 - Colar
- 7 - Mostrar 'cut buffer'
- 0 - Sair
- Escolha uma opção: Menu Principal
- 1 - Novo
- 2 - Abrir
- 3 - Guardar
- 4 - Menu de Edição
- 5 - Menu de Consultas
- 0 - Sair
  Escolha uma opção: 
\ No newline at end of file
--- 109,112 ----
