*** testsBothFinal/A-026-026-M-ok.out	2023-10-19 17:31:45.000000000 +0100
--- testsBothFinal/A-026-026-M-ok.outhyp	2023-10-26 06:54:57.828684575 +0100
***************
*** 5,12 ****
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: 4
! Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
--- 5,11 ----
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
***************
*** 15,46 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: 0
! Menu Principal
  1 - Novo
  2 - Abrir
  3 - Guardar
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: 5
! Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
! Escolha uma opção: 1
! Valor a procurar: 49
! 1;2|49
  Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
! Escolha uma opção: 0
! Menu Principal
  1 - Novo
  2 - Abrir
  3 - Guardar
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: 0
--- 14,44 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Menu Principal
  1 - Novo
  2 - Abrir
  3 - Guardar
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: Menu de Consultas
! 1 - Procurar valores
! 2 - Procurar funções
! 0 - Sair
! Escolha uma opção: Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
! Escolha uma opção: Opção inválida!
  Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
! Escolha uma opção: Menu Principal
  1 - Novo
  2 - Abrir
  3 - Guardar
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: 
\ No newline at end of file
