*** testsBothFinal/A-048-048-M-ok.out	2023-10-19 17:31:46.000000000 +0100
--- testsBothFinal/A-048-048-M-ok.outhyp	2023-10-26 06:26:16.385573988 +0100
***************
*** 5,12 ****
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: 4
! Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
--- 5,11 ----
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
***************
*** 15,24 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: 2
! Especifique a gama (startline;startcol:endline;endcol): 2;3
! Insira o conteúdo da célula: 10
! Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
--- 14,20 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Insira o conteúdo da célula: Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
***************
*** 27,35 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: 1
! Especifique a gama (startline;startcol:endline;endcol): 4;2
! 4;2|10=2;3
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 23,29 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 4;2|10=2;3
  Menu de Edição
  1 - Visualizar
  2 - Inserir
***************
*** 39,47 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: 1
! Especifique a gama (startline;startcol:endline;endcol): 2;3
! 2;3|10
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 33,39 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 2;3|10
  Menu de Edição
  1 - Visualizar
  2 - Inserir
***************
*** 51,62 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: 0
! Menu Principal
  1 - Novo
  2 - Abrir
  3 - Guardar
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: 0
--- 43,53 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Menu Principal
  1 - Novo
  2 - Abrir
  3 - Guardar
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: 
\ No newline at end of file
