<<<<<<< Updated upstream
*** testsBothFinal/A-081-081-M-ok.out	2023-10-19 17:31:47.000000000 +0100
--- testsBothFinal/A-081-081-M-ok.outhyp	2023-10-26 06:55:01.312772642 +0100
=======
*** testsBothFinal/A-081-081-M-ok.out	2023-10-26 05:36:05.590868673 +0100
--- testsBothFinal/A-081-081-M-ok.outhyp	2023-10-26 06:42:05.662397603 +0100
>>>>>>> Stashed changes
***************
*** 5,12 ****
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: 4
! Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
--- 5,11 ----
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
***************
*** 15,23 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: 3
! Especifique a gama (startline;startcol:endline;endcol): 2;1:2;3
! Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
--- 14,20 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
***************
*** 26,35 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: 7
! 1;1|25
  1;2|43
! 1;3|5=3;2
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 23,31 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: 1;1|25
  1;2|43
! 1;3|5
  Menu de Edição
  1 - Visualizar
  2 - Inserir
***************
*** 39,47 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: 6
! Especifique a gama (startline;startcol:endline;endcol): 5;1:5;3
! Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
--- 35,41 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
***************
*** 50,60 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: 1
! Especifique a gama (startline;startcol:endline;endcol): 5;1:5;3
! 5;1|25
  5;2|43
! 5;3|5=3;2
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 44,52 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 5;1|25
  5;2|43
! 5;3|5
  Menu de Edição
  1 - Visualizar
  2 - Inserir
***************
*** 64,75 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: 0
! Menu Principal
  1 - Novo
  2 - Abrir
  3 - Guardar
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: 0
--- 56,66 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Menu Principal
  1 - Novo
  2 - Abrir
  3 - Guardar
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: 
\ No newline at end of file
