<<<<<<< HEAD
*** testsFinal/A-038-038-M-ok.out	2023-10-26 05:36:05.594201986 +0100
--- testsFinal/A-038-038-M-ok.outhyp	2023-10-26 16:07:05.851941706 +0100
=======
*** testsFinal/A-038-038-M-ok.out	2023-10-19 17:15:15.000000000 +0100
<<<<<<< HEAD
<<<<<<< HEAD
<<<<<<< HEAD
<<<<<<< HEAD
--- testsFinal/A-038-038-M-ok.outhyp	2023-10-26 15:49:49.735083011 +0100
>>>>>>> main
=======
--- testsFinal/A-038-038-M-ok.outhyp	2023-10-26 16:09:56.930794747 +0100
>>>>>>> main
=======
--- testsFinal/A-038-038-M-ok.outhyp	2023-10-26 16:19:58.727027240 +0100
>>>>>>> main
=======
--- testsFinal/A-038-038-M-ok.outhyp	2023-10-26 17:25:31.186860878 +0100
>>>>>>> main
=======
--- testsFinal/A-038-038-M-ok.outhyp	2023-10-26 17:50:51.877260692 +0100
>>>>>>> main
***************
*** 14,20 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 2;3|#VALUE=AVERAGE(1;1:1;3)
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 14,20 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 2;3|18=AVERAGE(1;1:1;3)
  Menu de Edição
  1 - Visualizar
  2 - Inserir
***************
*** 36,42 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 4;1|#VALUE=PRODUCT(1;1:1;3)
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 36,42 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 4;1|0=PRODUCT(1;1:1;3)
  Menu de Edição
  1 - Visualizar
  2 - Inserir
