<<<<<<< HEAD
*** testsFinal/A-032-032-M-ok.out	2023-10-26 05:36:05.594201986 +0100
--- testsFinal/A-032-032-M-ok.outhyp	2023-10-26 16:07:05.528609092 +0100
=======
*** testsFinal/A-032-032-M-ok.out	2023-10-19 17:08:13.000000000 +0100
<<<<<<< HEAD
<<<<<<< HEAD
--- testsFinal/A-032-032-M-ok.outhyp	2023-10-26 15:49:49.235063249 +0100
>>>>>>> main
=======
--- testsFinal/A-032-032-M-ok.outhyp	2023-10-26 16:09:56.506777987 +0100
>>>>>>> main
=======
--- testsFinal/A-032-032-M-ok.outhyp	2023-10-26 16:19:58.287019654 +0100
>>>>>>> main
***************
*** 9,18 ****
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
! Escolha uma opção: Função a procurar: Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
  Escolha uma opção: Menu Principal
  1 - Novo
  2 - Abrir
--- 9,19 ----
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
! Escolha uma opção: Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Menu Principal
  1 - Novo
  2 - Abrir
