<<<<<<< HEAD
*** testsFinal/A-054-054-M-ok.out	2023-10-26 05:36:05.597535300 +0100
--- testsFinal/A-054-054-M-ok.outhyp	2023-10-26 16:07:06.731939747 +0100
=======
*** testsFinal/A-054-054-M-ok.out	2023-10-19 17:08:13.000000000 +0100
<<<<<<< HEAD
<<<<<<< HEAD
<<<<<<< HEAD
--- testsFinal/A-054-054-M-ok.outhyp	2023-10-26 15:49:51.079136132 +0100
>>>>>>> main
=======
--- testsFinal/A-054-054-M-ok.outhyp	2023-10-26 16:09:58.286848341 +0100
>>>>>>> main
=======
--- testsFinal/A-054-054-M-ok.outhyp	2023-10-26 16:19:59.951048384 +0100
>>>>>>> main
=======
--- testsFinal/A-054-054-M-ok.outhyp	2023-10-26 17:25:32.502838725 +0100
>>>>>>> main
***************
*** 25,31 ****
  0 - Sair
  Escolha uma opção: 1;1|15
  1;2|49
! 1;3|15=1;1
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 25,31 ----
  0 - Sair
  Escolha uma opção: 1;1|15
  1;2|49
! 1;3|15
  Menu de Edição
  1 - Visualizar
  2 - Inserir
